app/src/main/java/com/arielwang/workoutlogger/features/track/ui/screen/TrackViewModel.kt

```diff
@@ -11,12 +10,27 @@ import javax.inject.Inject
 
 object TrackView {
     data class State(
-        val notKnow: String = ""
+        var weightNumber: String = "0",
```

Comment:
Why this need a 0 as default value?

Reply:
Because the textField under the weight section need to show an initial value of 0

Reply:
The textField under the weight section needs to show an initial value of 0.